### ISSUE TITLE
not working.

### ISSUE THREAD

**Issue by radpfrarrer** [alexenferman/MStoreLinkGen#2]
try´ed like 5 different links and app´s did not get any results.
is ur service temporally disabled?

good job with your tutorials i relay liked them!

**Comment by alexenferman** [comment]
Yes you are correct, the proxy I was using now has stopped being free, therefore, the application is not working anymore and returning a 403 Forbidden error. I will have to find a new proxy and I will update you on this.
![image](https://user-images.githubusercontent.com/62704841/111793582-c87ee580-889b-11eb-8cd0-e84e3d4451b0.png)

**Comment by alexenferman** [comment]
Hello, the issue is now fixed. Enjoy and thanks for letting me know!

**Comment by lobotomy-x** [comment]
fyi its broken again either due to an expired token or due to not setting 'Access-Control-Allow-Origin' in your header. 

![image](https://github.com/alexenferman/MStoreLinkGen/assets/10409982/1dfc2ea7-4027-4d66-9cee-7c6350cb040e)
"Too Many Requests. This request is anonymous and rate limited. To upgrade, please add an api key at https://cors.sh"

I'm personally okay using the source website directly with trackers blocked but if you want to maintain this resource it will need some updates
